cli/detc.py

```diff
@@ -8,6 +8,27 @@ def cli():
   """Welcome to the DETC deployment tool"""
   pass
 
+@cli.command()
+@click.argument("cloud")
+@click.argument("action")
+def traffic(cloud, action):
+  """Lacework: Deploy Agents Pods
```

Comment:
Might want to add an example; I was trying to figure out how to run this and it wasn't clear it was `traffic <cloud> <action>`.  I know it's right in the usage string; but I did get stuck on it